### PR TITLE
Ee retrorun update

### PR DESCRIPTION
**Pull request by Langerz82**
Trying to get retrorun working for rockchip devices.

### CODE REVIEW

File: projects/Rockchip/packages/retrorun/package.mk [hunk 1]
```diff
@@ -2,32 +2,44 @@
...
 PKG_NAME="retrorun"
-PKG_VERSION="c83f1095351c82be9b04cdebd61ca0e03ef4cfb7"
+PKG_VERSION="7e3ab628d465a024a06ee113de50929b873157d0"
+PKG_ARCH="any"
 PKG_LICENSE="GPLv2"
-PKG_SITE="https://github.com/navy1978/retrorun-go2"
+PKG_SITE="https://github.com/AmberELEC/retrorun-go2"
 PKG_URL="$PKG_SITE.git"
-PKG_DEPENDS_TARGET="toolchain libgo2 libdrm"
+PKG_DEPENDS_TARGET="toolchain libgo2 libdrm libpng linux"
```
Comment (shantigilbert) [inline projects/Rockchip/packages/retrorun/package.mk:10]: linux is already included in the toolchain

File: projects/Rockchip/packages/retrorun/package.mk [hunk 2]
```diff
@@ -2,32 +2,44 @@
 # Copyright (C) 2021-present Shanti Gilbert (https://github.com/shantigilbert)
```
Comment (shantigilbert) [inline projects/Rockchip/packages/retrorun/package.mk:2]: Also please add your copyright information to the header (makes it easier to know who worked on which file)

**Comment by Langerz82** [comment]
I fixed the issues you mentioned.

@shantigilbert Would you know how I can create a 64-bit version and 32-bit version as well? I think handhelds would benefit using the 32-bit version (unless I am wrong).

**Comment by shantigilbert** [comment]
Retrorun is already compiled in 32bit mode <https://github.com/EmuELEC/emuelec-32bit-libs/tree/master/OdroidGoAdvance/usr/bin>

To compile it use `ARCH=arm ./scrips/build retrorun` but it will also compile the toolchain as well

**Comment by Langerz82** [comment]
@shantigilbert 
Is there a clean way I can build both 32-bit version and 64-bit version in package.mk and copy the separate output files to the image build?

**Comment by Langerz82** [comment]
@shantigilbert 
I think I understand the system in how the toolchain compiles a bit better. I have a solution where the user can just use an "ADD_ARM" argument followed by the packages they want compiled in ARM 32-bit. I'll have the code in a bit.

**Comment by shantigilbert** [comment]
I don't see the point of this, you seem build a lot of 32bit packages as well that are not needed in 32bit. That is why the emuelec-32bit-libs package is for.

**Comment by Langerz82** [comment]
what packages arent needed? I wanted to use it mainly for just retrorun 32-bit. Maybe just keep it as a draft and not commit to the main repo.

**Comment by shantigilbert** [comment]
Please separate the changes, retrorun changes by itself, and the ADD_ARM on another PR, altho I will tell you I will not merge it. 

Reasons: 

You are forcing the creation on a 32bit toolchain when building the image: 

`ARCH=arm ARCH_PARENT=$(ARCH) ./scripts/build_arm`

which will add a LOT more compilation time, hence why the 32bit-libs package exists.

If you want to add this for your own usage, then add it as an optional argument

example: 
instead of:

```
image:
	ARCH=arm ARCH_PARENT=$(ARCH) ./scripts/build_arm
	./scripts/image mkimage
```
add another entry on the makefile 


```
image:
	./scripts/image mkimage

image_with_arm:
	ARCH=arm ARCH_PARENT=$(ARCH) ./scripts/build_arm
	./scripts/image mkimage
```

**Comment by Langerz82** [comment]
@shantigilbert 
Sorry man I didn't mean to include the arm changes to the retrorun PR I must of been half asleep lol. Your changes sound good I will make the changes you said.

**Comment by Langerz82** [comment]
Im gonna test this on my handheld.

**Comment by Langerz82** [comment]
closing this as have new PR with clean changes.

**Pull request closed without merging**